### PR TITLE
Mod loading, dirs, and docs

### PR DESCRIPTION
**Pull request by ethn**

### CODE REVIEW

File: card/README.rdoc [hunk 1]
```diff
@@ -0,0 +1,103 @@
+
+=A Developer's introduction to the Decko/Card code base
+
+New folks should start here after playing with Wagn but before digging into code.
```
Comment (xithan) [inline card/README.rdoc]: Wagn?

File: card/README.rdoc [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+===Is Decko an _application_ or a <i>development framework</i>?
+
+It's both.
+
+When you install and seed a new Decko project, you have a working "deck" right away.  Your new application has lots of things built in: account handling, edit histories, default layouts, CSS, etc.
+
+But a newly seeded Deck is just a starting point.  Decko offers a rich framework within which to develop your application into a novel creation, both via configuration (as a "decker") or via code (as a "deck-coder").
+
+<b>Decko is a development framework in which you start <i>in the middle of things</i> rather than from scratch</b>.  Seeding is a bit of a headstart, and it means that designers, developers, and content creators can start working in parallel on day 1. But the seeded cards don't determine what you can build with Wagn; you can add things, you can remove things... In fact, you can build pretty much whatever you want.
```
Comment (xithan) [inline card/README.rdoc]: More Wagn.

**Comment by xithan** [comment]
Why are so many of my changes in the diff which I merged yesterday?

**Comment by ethn** [comment]
I don't get why those changes are there.  Maybe I should delete this PR and try again?

**Comment by xithan** [comment]
Worth a try and maybe merge master before try again.

**Comment by ethn** [comment]
diffs not showing.  will try again.

**Pull request closed without merging**